modules/table/src/main/java/org/apache/ignite/internal/utils/RebalanceUtil.java

```diff
@@ -468,7 +468,7 @@ public static CompletableFuture<Void> handleReduceChanged(MetaStorageManager met
      * @return Result of the subtraction.
      */
     public static <T> Set<T> subtract(Set<T> minuend, Set<T> subtrahend) {
```

Comment:
Can we get rid of this method?

Reply:
Not in this PR probably

Reply:
why? Is it used a lot?

Reply:
You are right, it's not a big deal. Removed it.